ajax/libs/react-extras/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "react-extras",
+  "title": "react-extras",
+  "description": "Useful components and utilities for working with React",
+  "author": {
+    "name": "Sindre Sorhus",
+    "url": "sindresorhus.com"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sindresorhus/react-extras"
+  },
+  "npmName": "react-extras",
+  "npmFileMap": [
+    {
+      "basePath": "source",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "index.min.js",
```

```suggestion
  "filename": "index.js",
```

Minified file doesn't exist in package, use the main `index.js` file that does exist.